packages/w/wayland-protocols/xmake.lua

```diff
@@ -0,0 +1,20 @@
+package("wayland-protocols")
+    set_homepage("Additional Wayland protocols")
+    set_description("Library for handling OpenGL function pointer management")
```

Comment:
Probably some confusion here between homepage/description 😄

Reply:
I have fixed it, thanks